src/targets/test.cpp

```diff
@@ -1,8 +1,11 @@
 #include <iostream>
 
 #include "Board.h"
+#include "Player.h"
 
 int main() {
+    //TODO waiting for constructor of Player
+    //Player white {}, black;
```

Comment:
Non serve più, l'ho aggiunto in #33. Puoi tranquillamente scartare le modifiche di `test.cpp`